--- Makefile	2015-04-12 18:38:18.000000000 +0200
+++ Makefile-patch	2015-04-12 17:59:46.000000000 +0200
@@ -88,7 +88,7 @@ lib${PROJECT}.a: ${OBJECTS}
 
 lib${PROJECT}.so.${SOVERSION}: ${OBJECTS}
 	$(call colorecho,LD,$@)
-	$(QUIET)${CC} -Wl,-soname,lib${PROJECT}.so.${SOMAJOR} -shared ${LDFLAGS} -o $@ ${OBJECTS} ${LIBS}
+	$(QUIET)${CC} -Wl,-install_name,lib${PROJECT}.so.${SOMAJOR} -shared ${LDFLAGS} -o $@ ${OBJECTS} ${LIBS}
 
 clean:
 	$(QUIET)rm -rf \
@@ -124,7 +124,7 @@ lib${PROJECT}-debug.a: ${DOBJECTS}
 
 lib${PROJECT}-debug.so.${SOVERSION}: ${DOBJECTS}
 	$(call colorecho,LD,$@)
-	$(QUIET)${CC} -Wl,-soname,lib${PROJECT}.so.${SOMAJOR} -shared ${LDFLAGS} -o $@ ${DOBJECTS} ${LIBS}
+	$(QUIET)${CC} -Wl,-install_name,lib${PROJECT}.so.${SOMAJOR} -shared ${LDFLAGS} -o $@ ${DOBJECTS} ${LIBS}
 
 debug: options ${PROJECT}-debug
 
