--- meson.build	2018-05-01 12:13:21.000000000 +0200
+++ meson.build	2018-03-16 16:41:57.000000000 +0100
@@ -36,7 +36,7 @@
 plugindir = join_paths(get_option('libdir'), 'zathura')
 
 # required dependencies
-libm = cc.find_library('libm')
+libm = cc.find_library('m')
 girara = dependency('girara-gtk3', version: '>=0.2.9')
 glib = dependency('glib-2.0', version: '>=2.50')
 gthread = dependency('gthread-2.0', version: '>=2.50')
